app/config/config_dev.yml

```diff
@@ -29,5 +29,5 @@ monolog:
         chromephp:
             type:   chromephp
             level:  info
-#swiftmailer:
-#    delivery_address: me@example.comdoctrine:
```

Comment:
Can you leave in this line commented out.  There will be a case where we want to test email delivery and it would be nice to be able to send it all to support.